composer.json

```diff
@@ -23,6 +23,9 @@
         "symfony/stopwatch": "^2.2|^3.0",
         "symfony/yaml": "^2.1|^3.0"
     },
+    "require-dev": {
```

Comment:
One more reason for: https://github.com/satooshi/php-coveralls/pull/147#issuecomment-163644696